src/main/java/seedu/clinic/commons/core/Messages.java

```diff
@@ -7,6 +7,7 @@
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n\n%1$s";
+    public static final String MESSAGE_INVALID_ALIAS = "Invalid Alias! \n\n%1$s";
```

Comment:
We might want to make "Alias" lowercase for consistency!

Reply:
Fair point. Thanks!